VATSpy.dat

```diff
@@ -17780,16 +17780,22 @@ EDGG-P|Langen Radar (Pad High) - Langen|EDGG_H|EDGG-P
 EDGG-P|Langen Radar (Pad High) - Langen|EDGG_P|EDGG-P
 EDGG-P|Langen Radar (Pad High) - Langen|EDGG_W|EDGG-P
 EDGG-P|Langen Radar (Pad High) - Langen|EDGG_NW|EDGG-P
-EDMM|Muenchen|EDMM_C|EDMM
-EDMM|Muenchen|EDMM_E|EDMM
-EDMM|Muenchen|EDMM_F|EDMM
-EDMM|Muenchen|EDMM_K|EDMM
-EDMM|Muenchen|EDMM_R|EDMM
-EDMM-G|Muenchen ACC (Gera) - Muenchen|EDMM_G|EDMM-G
-EDMM-G|Muenchen ACC (Gera) - Muenchen|EDMM_A|EDMM-G
-EDMM-G|Muenchen ACC (Gera) - Muenchen|EDMM_T|EDMM-G
-EDMM-R|Muenchen ACC (Roding) - Muenchen|EDMM_S|EDMM-R
+EDMM|Muenchen|EDMM_CTR|EDMM

```

Comment:
Remove EDMM_CTR. Its not possible to logon as EDMM_CTR_CTR

Reply:
Shouldnt it be EDMM|Muenchen|EDMM|EDMM ?

Reply:
fixed with next commit

Reply:
The one in between is only to define an alternative logon. If the FIR + logon matches, then you do not need to define it, so in this case both the FIR and logon is "EDMM".